tools/cldr-apps/js/src/views/DashboardWidget.vue

```diff
@@ -226,7 +226,7 @@ export default {
     };
   },
 
-  created() {
+  mounted() {
```

Comment:
Curious if this affects appearance, any delay/blinking/etc

Reply:
Mounted is after the dom is available